front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -282,7 +283,7 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
                     }
                     value={{
                       id: currentStudy.state,
-                      label: t(`studyStates.${currentStudy.state || 'nothingSelected'}`).toString(),
+                      label: t(`studyStates.${currentStudy.state || undefined}`).toString(),
```

Comment:
`currentStudy.state` is always defined, so you can remove the undefined :)
```suggestion
                      label: t(`studyStates.${currentStudy.state}`).toString(),
```